EasyPost/Base/Address.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace EasyPost.Base
+{
+    public abstract class Address : Resource
+    {
+        [JsonProperty("city")]
+        public string city { get; set; }
+        [JsonProperty("company")]
+        public string company { get; set; }
+        [JsonProperty("country")]
+        public string country { get; set; }
+        [JsonProperty("created_at")]
+        public DateTime? created_at { get; set; }
+        [JsonProperty("email")]
+        public string email { get; set; }
+        [JsonProperty("error")]
+        public string error { get; set; }
+        [JsonProperty("id")]
+        public string id { get; set; }
+        [JsonProperty("message")]
+        public string message { get; set; }
+        [JsonProperty("name")]
+        public string name { get; set; }
+        [JsonProperty("phone")]
+        public string phone { get; set; }
+        [JsonProperty("state")]
+        public string state { get; set; }
+        [JsonProperty("street1")]
+        public string street1 { get; set; }
+        [JsonProperty("street2")]
+        public string street2 { get; set; }
+        [JsonProperty("updated_at")]
+        public DateTime? updated_at { get; set; }
+        [JsonProperty("zip")]
+        public string zip { get; set; }
```

Comment:
It would be ideal to rearrange the order of these, Java for example: https://github.com/EasyPost/easypost-java/blob/master/src/main/java/com/easypost/model/BaseAddress.java#L6-L16

Reply:
They're in alphabetical order here, per our C# style guide.